### ISSUE TITLE
Merge stub style guide w/ official typing docs?

### ISSUE THREAD

**Issue by yangdanny97** [python/typeshed#12963]
I see that [contributing.md](https://raw.githubusercontent.com/python/typeshed/refs/heads/main/CONTRIBUTING.md) has a style guide for stubs, which somewhat duplicates the documentation on Writing and Maintaining Stubs on the [official typing docs](https://typing.readthedocs.io/en/latest/guides/writing_stubs.html). 

Should we reduce duplication by linking to the latter from the former? There might be a few extra typeshed-specific things, but for general style stuff I don't see why we need to write it all out again here.

**Comment by JelleZijlstra** [comment]
Yes, we should point people to the typing docs, and only add typeshed-specific guidance where it's different from the general guide.

**Comment by srittau** [comment]
In fact, the current style guide from the typing docs was originally copied from typeshed. But due to the limbo that the PEP – which ultimately became part of the docs – was in for a long time, the removal of the advice from the typeshed docs never happened. Most advice from CONTRIBUTING probably needs to be merged into the typing docs and then CONTRIBUTING needs slimming down.
